docs/src/modules/java/pages/views.adoc

```diff
@@ -498,4 +603,12 @@ To check if there are more pages left, you can use the function `has_more()` pro
 SELECT * AS customers, has_more() AS more_customers FROM customers LIMIT 10
 ----
 
-This query will return `more_customers = true` when the view contains more than 10 customers.
\ No newline at end of file
+This query will return `more_customers = true` when the view contains more than 10 customers.
+
+==== Total count of results
+
+To get the total number of results that will be returned over all pages, use `COUNT(*)` in a query that projects its results into a field. The total count will be returned in the aliased field (using `AS`) or otherwise into a field named `count`.
+
+----
+SELECT * AS customers, COUNT(*) AS total, has_more() AS more FROM customers LIMIT 10
+----
```

Comment:
do we need another example of COUNT(*) where it's not used together with paging?

Reply:
This special `COUNT(*)` is mainly for paging. It currently requires the `* AS results` projection into repeated field. Not much use for the total count outside of a query with a LIMIT.

I did mention in the original issue whether we support aggregating functions more generally.

Reply:
ok, so we don't support something like this?
```
SELECT COUNT(*) FROM customers
```

Reply:
Right, that's not currently supported.

Reply:
Created a separate issue for possibly extending this: https://github.com/lightbend/kalix-proxy/issues/1348